web/pnpm-lock.yaml

```diff
@@ -1,4 +1,8 @@
-lockfileVersion: '6.0'
+lockfileVersion: '6.1'
+
+settings:
+  autoInstallPeers: true
+  excludeLinksFromLockfile: false
```

Comment:
Was this something you changed? I would prefer if we skipped this part from this PR.

Reply:
I think it's because of a newer version of pnpm - we should probably add something to the `engines` section to ensure we all use the same version :)
```json
{
    "engines": {
        "node": ">=10",
        "pnpm": ">=3"
    }
}
```

Reply:
Ah yeah that might be it, could be smart to pin it to minor versions as well.

Reply:
Looks like https://github.com/pnpm/pnpm/releases/tag/v8.6.2 reverted the lockfile changes which is why we couldn't replicate the version bump locally (and in the other lockfiles).

So I'm just going to re-generate this lockfile and hopefully that should fix both the current issues and prevent future ones.